### ISSUE TITLE
In primevue-nuxt-module Button as NuxtLink not working

### ISSUE THREAD

**Issue by projct1** [primefaces/primevue#6038]
### Describe the bug

With module primevue-nuxt-module, the links should be converted to the `NuxtLink`, not `RouterLink`, which is logical.

### Reproducer

https://stackblitz.com/edit/nuxt-starter-fe8efa?file=app.vue

### PrimeVue version

4.0.0

### Vue version

3.x

### Language

ES6

### Build / Runtime

Nuxt

### Browser(s)

Any

### Steps to reproduce the behavior

https://stackblitz.com/edit/nuxt-starter-fe8efa?file=app.vue

![image](https://github.com/primefaces/primevue/assets/9752959/5203a8ad-bb5e-431a-bdf0-ac58e6a538f1)

### Expected behavior

Transforms to `NuxtLink` component.

**Comment by JPustkuchen** [comment]
Could we please extend this issue to all "Menu" components? It's the same issue. Any maintainer plans to fix this?

**Comment by mertsincan** [comment]
@projct1, sorry but I don't understand your screenshot. In stackblitz link, your first example uses NuxtLink and the other uses RouterLink. 🤔 Coming back to the main problem, Button uses `<component :is="as"` structure so, this is related to nuxtjs; https://github.com/nuxt/nuxt/issues/13659

@JPustkuchen, The menu components do not use any router APIs. You can integrate it using templating.

**Comment by JPustkuchen** [comment]
@mertsincan thanks for the quick reply! Yes, I saw it was removed. My question is - can it be done centrally?
Especially for the nuxt primevue module https://nuxt.com/modules/primevue it would be best to integrate the nuxt routing / `NuxtLink`. That's what I think Nuxt users would expect (and love ;))

Can it only be integrated by templating in each _instance_ of `<Menu>`, etc. or, maybe until it's part of the module, in any central place? (Maybe it could also be documented there, until integrated).

Sorry, I'm very new to PrimeVue and didn't find fitting documentation for this.
